app/views/portal/exhibitions.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+content = exhibition_content(@exhibition)
+json.exhibition_promo do
```

Comment:
this is the structure the frontend expects, it won't pick it up without this. I think we should leave it as is for now since the promo cards are due to be refactored anyway. Once that happens we can update this as well as remove some of the duplicate info(like 3 times the same image url being sent now).